agent/src/testflinger_agent/agent.py

```diff
@@ -311,7 +372,9 @@ def process_jobs(self):
                         # exit code 46 is our indication that recovery failed!
                         # In this case, we need to mark the device offline
                         if exit_code == 46:
-                            self.mark_device_offline()
+                            comment = "Set to offline by agent. Recovery "
+                            f"Failed found during {job.job_id} execution."
```

Comment:
I think this phrasing might be a little easier to read

```suggestion
                            comment = (
                                "Set to offline by agent. Recovery failed"
                                f" during job '{job.job_id}' execution."
                            )
```